pallets/pips/src/lib.rs

```diff
@@ -1345,12 +1340,12 @@ impl<T: Trait> Module<T> {
 
     /// Converts a PIP ID into a name of a PIP scheduled for execution.
     fn pip_execution_name(id: PipId) -> Vec<u8> {
-        Self::pip_schedule_name(&b"pip_execute"[..], id)
+        Self::pip_schedule_name(&PIP_EXECUTION[..], id)
```

Comment:
Could you instead move `pip_schedule_name` and `pip_expiry_name` to a more generic function in `pallets/common` that would compute a name given the prefix (preferably identified with an enum variant instead of a string constant) and a numeric index?

When this is done, you could use that generic name function in `pips`, `multisig` and `settlement` instead of the explicit pair & encode.

Reply:
I'll capture this in a new Jira story. We can do it in a different PR.